themes/hugo-book/layouts/partials/docs/html-head.html

```diff
@@ -8,7 +8,7 @@
 <link rel="canonical" href="{{ .Site.Params.CurrentBaseURL }}/{{ .File.Path | replaceRE "_index.md" "" | replaceRE "index.html.md" "" | replaceRE "_index.html" "" | replaceRE "README.md" "" |  replaceRE "readme.md" "" |  replaceRE ".md" ".html" |  replaceRE ".adoc" ".html" }}"/>
 <script src="{{ "javascripts/_gdpr-banner.js" | relURL }}" type="text/javascript"></script>
 <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
-
+<script src="{{ "javascripts/_img-zoom.js" | relURL }}" type="text/javascript"></script>
```

Comment:
This looks like it's a partial included in the top level JS.

Can we create a single JS file a bunch of includes in it, instead of loading from random URLs on the internet?